indigo_app/forms.py

```diff
@@ -354,8 +354,8 @@ def data_as_url(self):
 
 class WorkFilterForm(forms.Form):
     q = forms.CharField()
-    stub = forms.ChoiceField(choices=[('', 'Exclude stubs'), ('only', 'Only stubs'), ('temporary', 'Temporary stubs'), ('permanent', 'Permanent stubs'), ('all', 'Everything')])
-    status = forms.ChoiceField(choices=[('published', 'published'), ('draft', 'draft')])
+    stub = forms.MultipleChoiceField(choices=[('stub', 'Stub'), ('not_stub', 'Not stub'), ])
```

Comment:
I think `principal` below covers this now too

Reply:
note that we split out stubs into their own facet